tests/SpecTests/client-side-encryption/tests/badQueries.json

```diff
@@ -1387,7 +1387,7 @@
             }
           },
           "result": {
-            "errorContains": "Cannot encrypt element of type"
+            "errorContains": "Cannot encrypt element of type array"
```

Comment:
Similar to my comment below, these changes are an unintended reversion of the commit from [DRIVERS-2272](https://jira.mongodb.org/browse/DRIVERS-2272): https://github.com/mongodb/specifications/commit/6a7158d51b4c41f2f4a9c1293c5e1dceb93ab5c2. Those changes were applied in https://github.com/mongodb/mongo-php-library/commit/452bc0fe66a8b615787ac477b2514220e6d88434 (#916) for [PHPLIB-839](https://jira.mongodb.org/browse/PHPLIB-839).

Looking at the [history for `source/client-side-encryption/tests`](https://github.com/mongodb/specifications/commits/master/source/client-side-encryption/tests), the commit that updated `noSchema.json` (https://github.com/mongodb/specifications/commit/02b4275d4e53dcadf87682b9e07f08a86a7b81d5) is more recent than the previous commit the updated `badQueries.json` and `types.json` (https://github.com/mongodb/specifications/commit/6a7158d51b4c41f2f4a9c1293c5e1dceb93ab5c2).

@aleksandr-rudo How did you manage to pull in changes for `noSchema.json` and inadvertently revert these other two files at the same time?

Reply:
You mentioned that you actually checked out the specifications PR directly, which might explain it. The PR commit was https://github.com/mongodb/specifications/pull/1175/commits/4415510c598492b4b7ee5e9b4d13424064b4b802, which is five days older than the actual merge commit of https://github.com/mongodb/specifications/commit/02b4275d4e53dcadf87682b9e07f08a86a7b81d5.

https://github.com/mongodb/specifications/commit/6a7158d51b4c41f2f4a9c1293c5e1dceb93ab5c2, which modified `badQueries.json` and `types.json` appears between those two commit dates.

The downstream changes for DRIVERS tickets should always point to the merge commit (https://github.com/mongodb/specifications/commit/02b4275d4e53dcadf87682b9e07f08a86a7b81d5 in this case), which should avoid this kind of mistake.

Please revise this PR accordingly and stick to the direct commits referenced in downstream changes for DRIVERS tickets in the future.